.thoth.yaml

```diff
@@ -23,6 +23,7 @@ managers:
         - harshad16
         - pacospace
         - sesheta
+        - mayacostantini
```

Comment:
As this repo has an OWNERS file and the bot can obtain maintainer information from there, the suggestion here would be to remove the `maintainers` section from `.thoth.yaml` alltogether, and instead make sure that the list of `approvers` in the OWNERS file has the proper list:

https://github.com/thoth-station/report-processing/blob/master/OWNERS#L3

Reply:
It is done, thank you!